### DISCUSSION TITLE
App Unresponsive When Viewing Specific Card on Explore Page

### DISCUSSION THREAD

**Discussion by forchannot** [community/community#78367] [Mobile]
### Select Topic Area

Bug

### Body

Whenever I navigate to "pic1" on the explore page, the app consistently becomes unresponsive on my phone. The image in question resembles "pic2" when viewed on a computer. You can find the specific discussion at this link: [Discussion Link](https://github.com/babalae/better-genshin-impact/discussions/148).

I'm using a Xiaomi phone with Android version 13. The app version is 1.141.0, and I even tried the beta version, but unfortunately, it didn't resolve the issue.

<details>
  <summary>pic1</summary>
  <img src="https://github.com/community/community/assets/100580891/66701b5f-1af9-4e43-9a2c-18022a78e201">
</details>
<details>
  <summary>pic2</summary>
  <img src="https://github.com/community/community/assets/100580891/0aea89ba-cf5e-4794-9440-f0460550c437">
</details>

**Comment by stevepopovich** [comment]
Hi @forchannot thanks for writing. Yeah, that is a very large markdown file. I see it working well on newer devices, but it definitely has problems on older devices. I will pass this on to the team for consideration.
